src/framework/actions/hand.ts

```diff
@@ -112,6 +112,7 @@ export const playCard = (
     P.gold -= card.cost;
   }
 
+  P.discard.push(cid);
```

Comment:
This is actually incorrect; whether or not a card should go to discard immediately depends on the card type. For units (the only playable type atm, see below) and any other card that creates an instance, the card should be "tied up" in the instance until it leaves play. In fact, the only card type that goes immediately to discard is `INSTANT_SPELL`

Reply:
moved this down into the switch statement under `INSTANT_SPELL`